### ISSUE TITLE
3.0.8 Alias typo

### ISSUE THREAD

**Issue by McGreggor** [darkoperator/Posh-SSH#513]
Module alias is set to 'Get-SSHJsonKnowHost'  

When running Get-SSHTrustedHost I receive an error for unknown term 'Get-SSHJsonKnow**n**Host'

**Comment by MVKozlov** [comment]
this alias fixes typos existing in pre-3.0.8 (#484)
``` powershell
New-Alias -Name 'Get-SSHJsonKnowHost' -Value 'Get-SSHJsonKnownHost' -Force
```
and if you get unknown term '_Get-SSHJsonKnow**n**Host_' you are not using 3.0.8

**Comment by McGreggor** [comment]
I am on 3.0.8 and I can see the alias is set in scope, but I don't have a function named 'Get-SSHJsonKnownHost' that the alias points to in scope.  I can't find the function in 3.0.8's .ps1 file either.

**Comment by MVKozlov** [comment]
Here it is
https://github.com/darkoperator/Posh-SSH/blob/f00564d72ff566d446b9e858e89bff5c1a0a3013/Posh-SSH/Posh-SSH.psd1#L115
https://github.com/darkoperator/Posh-SSH/blob/f00564d72ff566d446b9e858e89bff5c1a0a3013/Source/PoshSSH/PoshSSH/NewSshJsonStore.cs#L7

please, try to `Get-Module Posh-SSH` and `Get-Module Posh-SSH -list` and look closely at the output

**Comment by McGreggor** [comment]
Yup, I have have checked Get-Module, and it shows 3.0.8.  I have ensured that specific version is imported in to scope via the `-RequiredVersion` parameter of the Import-Module cmdlet.  If I run `Get-Command -Module Posh-SSH -CommandType Cmdlet`  I see all the Cmdlets/Functions with 3.0.8 version number, but crucially there are two missing cmdlets when compared to those exported in the manifest:
```
Get-SSHJsonKnownHost
Get-SSHHostKey
```

Same result on two separate hosts.

**Comment by McGreggor** [comment]
Rolling back to 3.0.7, and doing the same check I can see `Get-SSHJsonKnowHost` in scope but 'Get-SSHHostKey` is missing from imported cmdlets despite being exported in the manifest.

**Comment by MVKozlov** [comment]
This is strange
what is your OS/Powershell versions ?

may you try to search all around for PoshSSH.dll ?
may be it loaded  from somewhere else ?

btw, this `get-command` variant give a lot of additional info
``` powershell
PS >Get-Command Get-SSHHostKey | fl *
...
DLL                 : ...\WindowsPowerShell\Modules\Posh-SSH\3.0.8\PoshSSH.dll
...
```
try it with `new-sshsession` for example

**Comment by McGreggor** [comment]
Running the `Get-Command | FL *` showed the DLL loaded was from v3.0.6.  That version had been loaded in scope first, but it seems despite successfully running a `Remove-Module` on it the DLL wasn't being unloaded, and nothing during the Import of 3.0.8 flagged it.  Having forced the uninstall of every version except 3.0.8 it now loads correct DLL and `Get-SSHTrustedHost` runs correctly.  
Thanks for your help.

**Comment by MVKozlov** [comment]
so, the issue can be closed for now?
